content/fr/ingress.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Ingress
+status: Completed
+category: technologie
+tags: ["fondamental"]
+---
+
+Un Ingress, que l'on pourrais traduire par "point d'entrée", est un ensemble de règles qui participe à gérer le trafic Internet depuis l'extérieur vers un conteneur ou un groupe de conteneurs s'exécutant dans un cluster.
```

Comment:
Dans nos règles d'écrite, le "nous" est préféré à "on".
"que nous pourrions traduire"

Reply:
fix in https://github.com/cncf/glossary/pull/2877/commits/906778cc4bacfa26d3bc213319d13920883a0a21